src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -251,6 +251,8 @@ public boolean link() throws IOException, InterruptedException {
 
         if (isAarch64) {
```

Comment:
it is actually a test on crosscompile (similar to the linkFlags)
If you're building on an AArch64 system itself, you don't have to specify a sysroot.

Reply:
ok, fixed